modules/installing-gitops-operator-in-web-console.adoc

```diff
@@ -10,10 +10,45 @@ You can install {gitops-title} Operator from the OperatorHub by using the web co
 
 .Procedure
 
-. Open the *Administrator* perspective of the web console and navigate to *Operators* → *OperatorHub* in the menu on the left.
+. Open the *Administrator* perspective of the web console and go to *Operators* -> *OperatorHub*.
 
 . Search for `OpenShift GitOps`, click the *{gitops-title}* tile, and then click *Install*.
+
+. On the *Install Operator* page:
+
+.. Select an *Update channel*.
+
+.. Select a {gitops-shortname} *Version* to install.
+
+.. Choose an *Installed Namespace*. The default installation namespace is `openshift-gitops-operator`.
```

Comment:
@Srivaralakshmi as you discovered: 
```
If i am installing gitops 1.9.3 ( any version prior to 1.10), am I supposed to see the `openshift-gitops-operator` 
namespace as the default installation namespace or `openshift-operators`? For ROSA, I am seeing only 
`openshift-operators` as the default installation namespace. Why is there an inconsistency in the experience? In 
the NOTE before the procedure, probably we have to mention this as an admonition for people who are 
upgrading to 1.10 from previous releases.
```
I checked this behavior again. With new versions, ROSA is acting the same/has the same issue. But I found out that the `openshift-gitops-operator` namespace is shown because of the chosen **update channel** being `gitops-1.10`. Once the user chooses an older channel, the namespace is no longer offered.

I found an IMPORTANT admonition here: [https://67805--docspreview.netlify.app/openshift-gitops/latest/installing_gitops/installing-openshift-gitops#prerequisites_install-gitops](https://www.google.com/url?q=https://67805--docspreview.netlify.app/openshift-gitops/latest/installing_gitops/installing-openshift-gitops%23prerequisites_install-gitops&sa=D&source=docs&ust=1700036530095818&usg=AOvVaw0O7dx7OgjEWQBa-WIk7ylx) that specifies the user should use the corresponding channel, so this might be an expected behavior, but still worth pointing out.